src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.important,
+.interesting,
+.surprising,
+div:last-child,
+div:first-child {
```

Comment:
how about just adding one more class and use with those elements you need?

Reply:
![image](https://user-images.githubusercontent.com/51531244/86224605-c18b3800-bb91-11ea-999f-09d5765125c6.png)

Reply:
In task, editing index.html are prohibited, or I misunderstood?

Reply:
adding new class will not break anything, but ok, in this case let it be as is